LSLib/Granny/Model/ColladaImporter.cs

```diff
@@ -710,12 +711,20 @@ private void ImportSkin(Root root, skin skin)
                     influenceSum += weights[weightIndex];
                 }
 
+                byte total = 0;
+                float remainder = 0;
+
                 for (var i = 0; i < influenceCount; i++)
                 {
                     var jointIndex = influences[offset + jointInputIndex];
                     var weightIndex = influences[offset + weightInputIndex];
                     var joint = joints[jointIndex];
-                    var weight = weights[weightIndex] / influenceSum;
+
+                    float weightF = weights[weightIndex] / influenceSum * 255 + remainder;
+                    byte weight = (byte)Math.Round(weightF);
+                    remainder = weightF - weight;
+                    total += weight;
```

Comment:
I'm not sure if this is the ideal solution but it seems to work pretty well. Compared to the previous behaviour, it'll at worst round a number up instead of down (or vice versa).